package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "org.montelibero.solar",
   "displayName": "Sunce Wallet",
-  "version": "1.0.0-beta.5",
+  "version": "1.0.0",
```

Comment:
iOS doesn't allow letters in `version`. see [here](https://github.com/Montelibero/mtl_solar/actions/runs/10425215471/job/28876085131#step:9:11)